pkg/api/v1/types.go

```diff
@@ -2048,7 +2048,7 @@ type NodeSpec struct {
 // DaemonEndpoint contains information about a single Daemon endpoint.
 type DaemonEndpoint struct {
 	// Port number of the given endpoint.
-	Port int32 `json:port`
+	Port int32 `json:"port"`
```

Comment:
Answering your question - it is, API server uses info in this when setting up proxy to the Node (which means proxy to the kubelet in practice).